### PR TITLE
Updated abbreviations for KilopoundForce on Force unit

### PR DESCRIPTION
**Pull request by dschuermans**
As per our discussion in https://github.com/angularsen/UnitsNet/pull/748#discussion_r376119040

I have updated the abbreviations for KilopoundForce in Force to "kipf", "kip" and "k"

### CODE REVIEW

File: Common/UnitDefinitions/Force.json [hunk 1]
```diff
@@ -156,11 +156,11 @@
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "kipf" ]
+          "Abbreviations": [ "kipf", "kip", "k" ]
         },
         {
           "Culture": "ru-RU",
-          "Abbreviations": [ "кипф" ]
+          "Abbreviations": [ "кипф", "койка", "К" ]
```
Comment (angularsen) [inline Common/UnitDefinitions/Force.json]: I'll just trust you on this one

Comment (dschuermans) [reply Common/UnitDefinitions/Force.json]: And i trusted google translate on this one :x

PS: Don't forget to release a new nuget? 😄

**Comment by angularsen** [comment]
[Release UnitsNet/4.45.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.45.0)

File: Common/UnitDefinitions/Force.json (hunk 1 above)
Comment (angularsen) [reply Common/UnitDefinitions/Force.json]: Forgot, thanks. Done!

**Pull request merged**